framework/generated/vulkan_generators/vulkan_state_table_header_generator.py

```diff
@@ -108,6 +114,14 @@ def generate_all(self):
             get_code += '    {0}* Get{0}(format::HandleId id) {{ return GetWrapper<{0}>(id, {1}); }}\n'.format(handle_wrapper, handle_map)
             const_get_code += '    const {0}* Get{0}(format::HandleId id) const {{ return GetWrapper<{0}>(id, {1}); }}\n'.format(handle_wrapper, handle_map)
             map_code += '    std::map<format::HandleId, {0}*> {1};\n'.format(handle_wrapper, handle_map)
+            vk_insert_code += '    bool InsertWrapper({0}* wrapper) {{ return InsertEntry(wrapper->handle, wrapper, {1}); }}\n'.format(handle_wrapper, handle_map)
+            vk_remove_code += '    bool RemoveWrapper(const {}* wrapper) {{\n'.format(handle_wrapper)
+            vk_remove_code += '         if (wrapper == nullptr) return false;\n'
+            vk_remove_code += '         return RemoveEntry(wrapper->handle, {});\n'.format(handle_map)
+            vk_remove_code += '    }\n'
+            vk_get_code += 'template<> inline {0}* VulkanStateHandleTable::GetWrapper<{0}>({1} handle) {{ return VulkanStateTableBase::GetWrapper(handle, {2}); }}\n'.format(handle_wrapper, vkhandle_name, handle_map)
+            vk_const_get_code += 'template<> inline const {0}* VulkanStateHandleTable::GetWrapper<{0}>({1} handle) const {{ return VulkanStateTableBase::GetWrapper(handle, {2}); }}\n'.format(handle_wrapper, vkhandle_name, handle_map)
+            vk_map_code += '    std::map<{0}, {1}*> {2};\n'.format(vkhandle_name, handle_wrapper, handle_map)
```

Comment:
Don't we want a `std::unordered_map`? I believe other projects like VL go further and use a robinhood map for even better performance.

Reply:
Thank you. Changed it.